frontend/rolecall/src/app/piece/piece_editor.component.ts

```diff
@@ -99,7 +100,9 @@ export class PieceEditor implements OnInit {
     this.displayedPieces = this.workingPieces.filter(piece => !piece.siblingId);
     this.displayedPieces.sort((a, b) => a.name < b.name ? -1 : 1);
     for (let i = 0; i < this.displayedPieces.length; i++) {
-      if (this.displayedPieces[i].isOpen) {
+      if (this.displayedPieces[i].type === 'SUPER' &&
```

Comment:
Referencing `this.displayedPieces[i]` four times, it'll be a little more readable if you first set
```
const piece = this.displayedPieces[i];
```
and use that in the four places

Reply:
Done.